src/index.js

```diff
@@ -116,7 +118,19 @@ function ParseServer({
   const filesController = new FilesController(filesControllerAdapter);
   const pushController = new PushController(pushControllerAdapter);
   const loggerController = new LoggerController(loggerControllerAdapter);
-  
+
+  if (verifyUserEmails) {
+    if (typeof appName !== 'string') {
```

Comment:
Not 100% sure I follow on this either. It validates the adapter when the MailController/UserController is instantiated?